File: 7-kyu/find-the-capitals-1/find-the-capitals-1.py
def capitals(word):
    #your code here
    # create empty list
    word_list = []
    # turn word into list
    for l in word:
        word_list.append(l)
    
    new_list = []
    
    for i, l in enumerate(word_list):
        if l == l.upper():
            new_list.append(i)
        else:
            continue
    return new_list
        

File: 7-kyu/find-the-capitals-1/find-the-capitals-1.js
var capitals = function (word) {
  // Write your code here
  let capitalList = [];
  for (let i = 0; i < word.length; i++) {
    let char = word[i];
    if (char === char.toUpperCase() && char !== char.toLowerCase()) {
      capitalList.push(i);
    } else {
      continue;
    }
  } 
  return capitalList;
};